apps/explorer/components/Host/HostSettings.tsx

```diff
@@ -15,17 +14,19 @@ import {
   getStorageCost,
   getUploadCost,
 } from '@siafoundation/units'
-import { useExchangeRate } from '../../hooks/useExchangeRate'
-import { siacoinToFiat } from '../../lib/currency'
 import { ExplorerHost } from '@siafoundation/explored-types'
+import { useActiveCurrencySiascanExchangeRate } from '@siafoundation/react-core'
+import FiatDisplay from '../FiatDisplay'
+import LoadingCurrency from '../LoadingCurrency'
 
 type Props = {
   host: ExplorerHost
-  rates?: SiaCentralExchangeRates
 }
 
-export function HostSettings({ host, rates }: Props) {
-  const exchange = useExchangeRate(rates)
+export function HostSettings({ host }: Props) {
+  const exchange = useActiveCurrencySiascanExchangeRate({
+    api: 'https://api.beta.siascan.com/api',
```

Comment:
We should move this string out to the config file so its easy to swap all instances when we release. Also, if we include the ssr changes we can use the variable for the fallback key as well.